Yoast/yoast-components
# Add Synonyms component

## Summary

This PR can be summarized in the following changelog entry:

* Adds synonyms components.

## Relevant technical choices:

* Adds SynonymsSection and SynonymsInput components.
* Adds a synonym redux action and reducer.

## Test instructions

This PR can be tested by following these steps:

* Check the standalone under the tab "Keyword".
* Type something in the synonyms input and see the console log.

Fixes #622
